uv.gyp

```diff
@@ -274,19 +274,43 @@
           },
         }],
         [ 'OS=="aix"', {
-          'sources': [ 'src/unix/aix.c' ],
+          'variables': {
```

Comment:
Visually, there seem to be more differences than commonalities in this section between aix and OS400, I think the os400 section should be separate from aix maybe!
```gyp
[ 'OS=="aix"', {
   ...
}],
[ 'OS=="os400"', {
   ...
}],
```

Reply:
Currently the IBM i python returns "aix6" for sys.platform. It's great when everything in PASE works the same in AIX (the normal conditions), but in some rare conditions (like libuv), we have to handle it specifically.